experimental/python/codegen/.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
Is there a specific reason for requiring 3.13?

Reply:
Yeah, agree. Can we use one of `pyproject.toml` or this file instead of both?

Reply:
Makes sense, fixed